source/Octopus.Tentacle/Configuration/Instances/ApplicationInstanceStore.cs

```diff
@@ -92,16 +92,38 @@ public IList<ApplicationInstanceRecord> ListInstances()
             var listFromRegistry = registryApplicationInstanceStore.GetListFromRegistry();
             var listFromFileSystem = new List<ApplicationInstanceRecord>();
             if (fileSystem.DirectoryExists(InstancesFolder))
```

Comment:
i think we ought to have a env var which lets us control the instance folder so each test can be isolated.

Reply:
I would prefer we do this over swallowing exeptions, but approved if u want to proceed as is

Reply:
Mmm, that would be better 👍